util.go

```diff
@@ -11,3 +24,289 @@ func slice2LowerMap(array []string) map[string]bool {
 
 	return out
 }
+
```

Comment:
Can you move the getpass to a new file like getpass.go? that way it can be updated easily in the future if needed.

Reply:
done